_tests/integration/cordova_test.go

```diff
@@ -99,10 +101,12 @@ configs:
           - git-clone@%s: {}
           - script@%s:
               title: Do anything with Script step
+          - restore-npm-cache@%s: {}
```

Comment:
Wouldn't it be better if cache-restore directly follows git-clone?

Reply:
Wouldn't it be even better to remove the script step? 😅 AFAIK we agreed to remove all of them, Cordova seems to be a leftover.

Reply:
Fixed